spec/requests/api/merge_requests_spec.rb

```diff
@@ -108,6 +108,14 @@
       json_response['iid'].should == merge_request.iid
     end
 
+    it "should return merge_request by iid" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.